integ-test/src/test/resources/indexDefinitions/weblogs_index_mapping.json

```diff
@@ -1,9 +1,12 @@
 {
   "mappings": {
     "properties": {
-      "host": {
+      "host_ip": {
         "type": "ip"
       },
+      "host_string": {
+        "type": "keyword"
+      },
       "method": {
         "type": "text"
```

Comment:
Split into to fields with IP address and string data types, respectively. Should facilitate testing both once we add support for IP address data type in #3145.